### PR TITLE
Create Reid_Ferguson_2011

### PR DESCRIPTION
**Pull request by MarleneDH**
Reid DE, Ferguson BJ, Gresshoff PM. Inoculation- and nitrate-induced CLE peptides of soybean control NARK-dependent nodule formation. Mol Plant Microbe Interact. 2011 May;24(5):606-18. doi: 10.1094/MPMI-09-10-0207. PMID: 21198362.

Any feedback and suggestions would be much appreciated!

### CODE REVIEW

File: Glycine/max/wip_marlene/Reid_Ferguson_2011 [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  - citation: Reid, Ferguson et al., 2011
+    doi: 10.1094/MPMI-09-10-0207
+    pmid: 21198362
+ ---
+ gene_symbols:
+  - RIC2
+  - GmRIC2
+gene_symbol_long: rhizobia-induced CLE
+gene_model_pub_name: Glyma.06g43680
+gene_model_full_id: NOT FOUND
```
Comment (MarleneDH) [inline Glycine/max/wip_marlene/Reid_Ferguson_2011:29]: I'm having trouble finding the gene ID for Glyma.06g43680.

Comment (MarleneDH) [reply Glycine/max/wip_marlene/Reid_Ferguson_2011:29]: I searched for GmRIC1 in the ncbi database and found a different pub name than the one in the paper: Glyma.13G292300.  It sounds like the same gene based on the description and it does have an ID: glyma.Wm82.gnm2.ann1.Glyma.13G292300

File: Glycine/max/wip_marlene/Reid_Ferguson_2011 [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  - citation: Reid, Ferguson et al., 2011
+    doi: 10.1094/MPMI-09-10-0207
+    pmid: 21198362
+ ---
+ gene_symbols:
+  - RIC2
+  - GmRIC2
+gene_symbol_long: rhizobia-induced CLE
+gene_model_pub_name: Glyma.13G292300
+gene_model_full_id: glyma.Wm82.gnm2.ann1.Glyma.13G292300
```
Comment (MarleneDH) [inline Glycine/max/wip_marlene/Reid_Ferguson_2011:29]: Update: I searched for GmRIC1 in the ncbi database and found a different pub name than the one in the paper: Glyma.13G292300. It sounds like the same gene based on the description and it does have an ID: glyma.Wm82.gnm2.ann1.Glyma.13G292300

**Review by StevenCannon-USDA — APPROVED** [review]
Looks OK. Will make minor changes if necessary after merge.

**Pull request merged**